Description: Add opusenc support needed by Telegram app (LP: #1630399)
Author: Devid Antonio Filoni <d.filoni@ubuntu.com>
Bug: https://bugreports.qt.io/browse/QTBUG-50567
Bug-Ubuntu: https://bugs.launchpad.net/bugs/1630399
Reviewed-By: Alfonso Sanchez-Beato <alfonso.sanchez-beato@canonical.com>

--- qtmultimedia-opensource-src-5.6.1.orig/src/plugins/gstreamer/mediacapture/qgstreameraudioencode.cpp
+++ qtmultimedia-opensource-src-5.6.1/src/plugins/gstreamer/mediacapture/qgstreameraudioencode.cpp
@@ -47,10 +47,10 @@ QGstreamerAudioEncode::QGstreamerAudioEn
 
 #if defined(Q_WS_MAEMO_6)
     codecCandidates << "audio/AAC" << "audio/mpeg" << "audio/vorbis" << "audio/speex" << "audio/GSM"
-                    << "audio/PCM" << "audio/AMR" << "audio/AMR-WB" << "audio/FLAC";
+                    << "audio/PCM" << "audio/AMR" << "audio/AMR-WB" << "audio/FLAC" << "audio/x-opus";
 #else
     codecCandidates << "audio/mpeg" << "audio/vorbis" << "audio/speex" << "audio/GSM"
-                    << "audio/PCM" << "audio/AMR" << "audio/AMR-WB" << "audio/FLAC";
+                    << "audio/PCM" << "audio/AMR" << "audio/AMR-WB" << "audio/FLAC" << "audio/x-opus";
 #endif
 
 #if defined(Q_WS_MAEMO_6)
@@ -68,6 +68,7 @@ QGstreamerAudioEncode::QGstreamerAudioEn
     m_elementNames["audio/PCM"] = "audioresample";
     m_elementNames["audio/FLAC"] = "flacenc";
     m_elementNames["audio/GSM"] = "gsmenc";
+    m_elementNames["audio/x-opus"] = "opusenc";
 
     m_codecOptions["audio/vorbis"] = QStringList() << "min-bitrate" << "max-bitrate";
     m_codecOptions["audio/mpeg"] = QStringList() << "mode";
@@ -76,6 +77,7 @@ QGstreamerAudioEncode::QGstreamerAudioEn
     m_codecOptions["audio/PCM"] = QStringList();
     m_codecOptions["audio/AMR"] = QStringList();
     m_codecOptions["audio/AMR-WB"] = QStringList();
+    m_codecOptions["audio/x-opus"] = QStringList();
 
     for (const QByteArray& codecName : qAsConst(codecCandidates)) {
         QByteArray elementName = m_elementNames[codecName];
